crates/bevy_ui/src/geometry.rs

```diff
@@ -356,39 +355,39 @@ impl Size {
     };
 }
 
-impl Add<Vec2> for Size {
+impl Add<(Val, Val)> for Size {
     type Output = Size;
 
-    fn add(self, rhs: Vec2) -> Self::Output {
+    fn add(self, rhs: (Val, Val)) -> Self::Output {
```

Comment:
I think you should add doc with a "panic" section to mention cases this method may panic.

Same for the other operator methods.